.azure/build.yaml

```diff
@@ -56,25 +56,25 @@ steps:
           --name "$(cdn_endpoint)" \
           --resource-group "$(resource_group)"
   
-  - task: AzureCLI@2
-    displayName: Download version file from Azure Storage Account and set variable
-    inputs:
-      azureSubscription: $(resource_group)
-      scriptType: bash
-      scriptLocation: inlineScript
-      inlineScript: |
-        echo "[INFO] downloading version file from Azure Storage Account"
-        az storage blob download \
-          --account-name "$(storage_account_name)" \
-          -c "choreo-samples" \
-          -n "choreo-samples-version.txt" \
-          -f "$(Build.ArtifactStagingDirectory)/choreo-samples-version.txt" \
-          --overwrite true
-        echo "[INFO] reading samples version from of the version file"
-        SAMPLES_VERSION=$(< "$(Build.ArtifactStagingDirectory)/choreo-samples-version.txt")
-        echo "##vso[task.setvariable variable=SAMPLES_VERSION]$SAMPLES_VERSION"     
+  # - task: AzureCLI@2
+  #   displayName: Download version file from Azure Storage Account and set variable
+  #   inputs:
+  #     azureSubscription: $(resource_group)
+  #     scriptType: bash
+  #     scriptLocation: inlineScript
+  #     inlineScript: |
+  #       echo "[INFO] downloading version file from Azure Storage Account"
+  #       az storage blob download \
+  #         --account-name "$(storage_account_name)" \
+  #         -c "choreo-samples" \
+  #         -n "choreo-samples-version.txt" \
+  #         -f "$(Build.ArtifactStagingDirectory)/choreo-samples-version.txt" \
+  #         --overwrite true
+  #       echo "[INFO] reading samples version from of the version file"
+  #       SAMPLES_VERSION=$(< "$(Build.ArtifactStagingDirectory)/choreo-samples-version.txt")
+  #       echo "##vso[task.setvariable variable=SAMPLES_VERSION]$SAMPLES_VERSION"     
 
-  - template: update-image-in-cp-overlays.yml@common-templates
-    parameters:
-      CONFIGS:
-        - CHOREO_SAMPLES_VERSION='$(SAMPLES_VERSION)'
+  # - template: update-image-in-cp-overlays.yml@common-templates
+  #   parameters:
+  #     CONFIGS:
+  #       - CHOREO_SAMPLES_VERSION='$(SAMPLES_VERSION)'
```

Comment:
we need this commments?

Reply:
Skipping these steps only for this PR. Need to add them again for the implementation.